CameraOverlay/configs.json

```diff
@@ -1,13 +1,3 @@
 {
-  "activeOverlay": "Overlay_Demo2.py",
-  "overlays": [
-    {
-      "file": "Overlay_Demo.py",
-      "name": "My first overlay"
-    },
-    {
-      "file": "Overlay_Demo2.py",
-      "name": "My second overlay"
-    }
-  ]
-}
\ No newline at end of file
+  "activeOverlay": "Overlay_Demo2.py"
```

Comment:
Now that `activeOverlay` is the only thing stored in `configs.json` in the repo, should we remove this file from the repo (and .gitignore it) and instead have instructions to create it in the readme? That way we won't have conflicts when we `git pull` and have made changes to this file. We could always add a fallback overlay in code if this is not set.

Reply:
That's a good idea. We should do this on another PR though!

Reply:
i think since we have an .env file, we might as well remove configs.json and just have the ACTIVE_OVERLAY as an environment variable?

what do we think? @hallgchris @khanguslee

Reply:
No. I don't think we should set environment variables dynamically. I just don't think it's right even though it does work.

Reply:
Agreed, especially considering that would mean having the code modifiy the .env file

Reply:
fair point. alright, i'll update the readme and do the needful ;)